authz/build.gradle

```diff
@@ -0,0 +1,17 @@
+plugins {
```

Comment:
would it make sense to move entities/auth as well into this module ?

Reply:
if modules are owning their entities then it would make the entities module redundant

Reply:
agree for Varadhi domain entities, but entities/auth has enum like resource types and resource actions and config which seems different from other entities. It has no intended uses in other modules (except authz and server).

Reply:
done